app/main.py

```diff
@@ -1,3 +1,48 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance | int | float) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        else:
```

Comment:
u can ignore else here!

Reply:
i can't, just tried to do that and it returned 10 mistakes
![image](https://user-images.githubusercontent.com/123902666/218396039-bc16b8b4-386f-4ddd-9ef6-31933e27455d.png)